platform/resolve_inputs.go

```diff
@@ -20,6 +20,12 @@ var (
 )
 
 func ResolveInputs(phase LifecyclePhase, i *LifecycleInputs, logger log.Logger) error {
+	if i.UseDaemon || i.UseLayout {
```

Comment:
Is `LocalimageStrategy` correct for Daemon run images in all cases?

I thought `pack build <blah> --publish` , for instance, would read the run image from the target registry to place the app layers onto without pulling it down.

Reply:
Ah yes, but in the `--publish` case `i.UseDaemon` would be false

Reply:
According to [this comment](https://github.com/buildpacks/lifecycle/pull/1024#issuecomment-1476360242)

> If the export target is the daemon, then we also expect to find the run image in a daemon. We don't allow "mixing and matching" although maybe (someday) we should.

But does `--publish` not imply that you export to a registry and not the daemon?

Reply:
As long as `i.UseDaemon` is false I think we are good.